disko-install

```diff
@@ -110,6 +110,15 @@ parseArgs() {
       extraSystemConfig="$2"
       shift
       ;;
+    --disko-system-config)
```

Comment:
Should be part of `usage` with some explanation, since it's not obvious what it's useful for.